data/wp/wp-content/plugins/epfl/shortcodes/epfl-people/epfl-people.php

```diff
@@ -45,6 +45,12 @@ function epfl_people_2018_process_shortcode( $attributes, $content = null )
 
   ("" !== $units) ? $parameter['units'] = $units : $parameter['scipers'] = $scipers;
 
+  if (get_locale() == 'fr_FR') {
```

Comment:
Si ton but est de savoir dans quelle langue l'utilisateur est en train de voir le site, ça ne va pas marcher avec ça. Parce que `get_locale()` renvoie la langue configurée pour l'administration du site... 
Il faudrait donc utiliser le même code que celui dans le thème (`header.php`) afin d'utiliser une fonction Polylang pour récupérer la langue et si celui-ci n'est pas installé, avoir une langue par "défaut".

> $default_lang = 'en';
$allowed_langs = array('en', 'fr');
$parameter['lang'] = $default_lang;
/* If Polylang installed */
		if(function_exists('pll_current_language'))
		{
			/* Get current lang */
			 $current_lang = pll_current_language('slug');
			 /* Check if current lang is supported. If not, use default lang*/
			 $parameter['lang'] = (in_array($current_lang, $allowed_langs))?$current_lang:$default_lang;
		}

Reply:
arf, bien vu, j'avais oublié polylang. Je corrige.